kubernetes/csmsshkeys/Chart.yaml

```diff
@@ -1,5 +1,30 @@
-apiVersion: v1
+annotations:
+  artifacthub.io/images: |
+    - name: cfs-trust
```

Comment:
I'm guessing this name should be csm-ssh-keys

Reply:
You have guessed incorrectly.

Reply:
Correct! Updated.